### ISSUE TITLE
Add a camera fails without giving useful feedback message

### ISSUE THREAD

**Issue by mosic** [evercam/evercam-dashboard#1092]
If the request to the API fails with an `unknown_error` the dashboard is displaying a general error message ("An error occurred processing your request. Please try again and if the problem persists contact support.") even though the API returns a useful error message.

One example, this is the output from API logs when a user tries to add a camera with a name longer than 24 characters:

```
Evercam error caught processing request.
Type: Evercam::OutcomeError
Code: 
HTTP Status Code: 400
Context: []
Stack Trace:
/app/app/api/v1/routes/camera_routes.rb:279:in `block (2 levels) in <class:V1CameraRoutes>'
/app/vendor/bundle/ruby/2.2.0/gems/grape-0.13.0/lib/grape/endpoint.rb:51:in `call'
/app/vendor/bundle/ruby/2.2.0/gems/grape-0.13.0/lib/grape/endpoint.rb:51:in `block (2 levels) in generate_api_method'
/app/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.6/lib/active_support/notifications.rb:166:in `instrument'
/app/vendor/bundle/ruby/2.2.0/gems/grape-0.13.0/lib/grape/endpoint.rb:50:in `block in generate_api_method'
I, [2016-04-06T21:51:11.707205 #10]  INFO -- : Response:
{
  "message": "Camera Name is too long. Maximum 24 characters.",
  "code": "unknown_error",
  "context": [
    "name"
  ]
}
```

**Comment by liutingdu** [comment]
In Android and iOS is displaying the value of the "message", no matter what the response code is.

**Comment by marcoherbst** [comment]
@liutingdu you're great.

**Comment by muneeb2542** [comment]
This issue is fixed in #1093 

Closing this.
